src/components/interaction-designer/block-editors/choices/SelectOneResponseBlockContactPropertyEditor.vue

```diff
@@ -0,0 +1,178 @@
+    <generic-contact-property-editor
+      :block="block"
+      @toggleSetContactProperty="onSetContactPropertyToggle"
+      @changeContactPropertyType="resetMapping"
+      @updateShouldUseCurrentBlockResponse="onShouldUseCurrentBlockResponseUpdate">
+      <template slot="entry-from-this-block">
+        <div class="mt-2 mb-2">
+          <div
+            v-if="choiceKeys.length === 0"
+            class="mt-2 mb-2 text-danger">
+            {{ trans('flow-builder.enter-at-least-one-choice-above') }}
+          </div>
+          <template v-else>
+            <div v-if="isMultipleChoiceProperty">
+              {{ trans('flow-builder.select-value-for-choices-for-selected-property') }}
+            </div>
+            <div v-else>
+              {{ trans('flow-builder.enter-value-for-choices-for-selected-property') }}
+            </div>
+
+            <div
+              v-for="choiceKey in choiceKeys"
+              :key="choiceKey"
+              class="mt-2">
+              <label>{{ trans('flow-builder.choice') }}: {{ choiceKey }}</label>
+              <vue-multiselect
+                v-if="isMultipleChoiceProperty"
+                :value="getChoiceValueOption(choiceKey)"
+                :options="choiceValueOptions"
+                :show-labels="false"
+                :placeholder="trans('flow-builder.select-a-value')"
+                track-by="value"
+                label="description"
+                @input="setChoiceValueOption($event, choiceKey)" />
+              <input
+                v-if="isTextProperty"
+                :value="getChoiceValue(choiceKey)"
+                type="text"
+                class="form-control"
+                :placeholder="trans('flow-builder.enter-value')"
+                @input="setChoiceValue($event.target.value, choiceKey)">
+              <input
+                v-if="isNumberProperty"
+                :value="getChoiceValue(choiceKey)"
+                type="number"
+                class="form-control"
+                :placeholder="trans('flow-builder.enter-value')"
+                @input="setChoiceValue(Number($event.target.value), choiceKey)">
+            </div>
+          </template>
+        </div>
+      </template>
+    </generic-contact-property-editor>
+  </div>
+</template>
+
+<script lang="ts">
+import VueMultiselect from 'vue-multiselect'
+import Component, {mixins} from 'vue-class-component'
+import {Prop} from 'vue-property-decorator'
+import Lang from '@/lib/filters/lang'
+import {ISelectOneResponseBlock} from '@floip/flow-runner/src/model/block/ISelectOneResponseBlock'
+import {IContactPropertyOption} from '@/store/flow/block-types'
+import {Getter, namespace} from 'vuex-class'
+import {find} from 'lodash'
+import {IContactPropertyMultipleChoice} from '@/store/flow/block-types/Core_SetContactPropertyStore'
+import {choicesToExpression} from './expressionTransformers'
+
+const flowVuexNamespace = namespace('flow')
+
+@Component({
```

Comment:
Let's use option API as discussed with devX subteam

Reply:
🤬, I forgot to push my changes, sorry

Reply:
Ah ok, thanks